fill_esd_log.py

```
@@ -418,85 +429,203 @@ def main() -> None:
+
+                        contact_span.set_attribute(
+                            "contact.business_name", business_name
+                        )
+
+                        try:
+                            contact_info = contact_info_service.get_contact_info(
+                                c["business_name"]
+                            )
+
+                            if "error" in contact_info:
+                                contact_span.set_attribute("enrichment.success", False)
```

Ruff (0.14.2)

Boolean positional value in function call

(FBT003)